src/app/sb-admin/service/user.service.ts

```diff
@@ -30,6 +30,6 @@ export class UserService {
       "Authorization": environment.authKey,
       "x-authenticated-user-token": environment.userToken,
     })
-    return this.http.post(environment.userRoleUrl, payload, { headers: header });
+    return this.http.post(environment.userCreateUrl, payload, { headers: header });
```

Comment:
use the correct naming convention for update role url

Reply:
this url for create new user,and userRole is in other PR